aws/v19.1.0/README.md

```diff
@@ -0,0 +1,214 @@
+
+Feature can be enabled via an annotation `cilium.giantswarm.io/ipam-mode: eni` set on the Cluster CR while on `18.4.2` release prior to `19.1.0` upgrade. When the upgrade is triggered, the undelying infrastructure will choose to continue with the [Cilium AWS ENI mode](https://docs.cilium.io/en/latest/network/concepts/ipam/eni/). This is meant for the users that do not want to migrate any of the underlying network infrastructure that has been linked with the Giant Swarm Workload Clusters. The network setup after the upgrade will be the same as while running `aws-cni` with kube-proxy.
+
+## Kyverno by default
+
+This is the release preparing for the migration away from Pod Security Policies (PSP) in favor of Pod Security Standards (PSS) in Kubernetes 1.25. Our `security-bundle` is now installed by default, and will deploy `kyverno` and `restricted` level PSS policies in `audit` mode. These resources are provided in order to allow time to create exceptions for workloads which need them before the policies are changed to `enforce` in a future release. For more information about PSS please read our official [documentation](https://docs.giantswarm.io/advanced/security-policy-enforcement/). Please also take a look at the `kyverno` [documentation](https://docs.giantswarm.io/platform-overview/security/platform-security/#kyverno) to utilize fully its potential.
+
+> **WARNING:** If you are already running `kyverno` as Giant Swarm Managed App, the installation of `security-bundle` will fail. However the already existing `kyverno` deployment and its configuration can be adopted by the bundle after the upgrade is finished. Please talk to your Account Engineer if you have any questions.
+
+## AWSMachineDeployment CR's annotation to change the Flatcar Release Version
+
+This feature allows customer to set an annotation on AWSMachineDeployment CR's to change the Flatcar Release Version. For now it only allows to set `alpha.giantswarm.io/flatcar-release-version: "3689.0.0"` or a higher. We have added this feautre to accomodate the issues with Cilium CNI high CPU usage on small clusters. This feature is solely to enable customers to run Flatcar `alpha` channel which consists of the `kernel 6` version that fixes the issue, while waiting for a `stable` Flatcar release.
+
+Please read the detailed description of the designed behaviour of the annotation:
+- when setting the annotation the TCNP Stack for the specific node pool is rolled and replaces the OS image
+- when removing the annotation the node pool is updated and switches back to the default OS image which is coming from the AWS release
+- when upgrading the cluster to a new AWS release, the node pool uses the specific flatcar release from the annotation as long as you don't change by either setting it to a higher version or removing the annotation.
```

Comment:
What if a customer fixed the Flatcar version to (imagine) 1.23.4 while the new AWS release requires 1.34.5, is not compatible with the former and would break the cluster during upgrade?

Reply:
You can only take the default flatcar version from the release or upgrade to 3689.0.0 or higher. However we have a JSON file for choosing the right AMI for every AWS region. If the flatcar version isn't found you can't to upgrades, meaning you can't accidentally select a wrong flatcar release

Reply:
You might have gotten me wrong: Imagine being on a AWS release and having defined your own Flatcar version for a node pool. You then want to upgrade to another AWS release and still have defined your own Flatcar version in the node pool, but the new AWS release is not compatible with this Flatcar version.

Are we then falling back to the default Flatcar version shipped with the new AWS release or is the whole upgrade being stopped/prevented?

Reply:
It will have to be reviewed before every upgrade, what Flatcar version there is etc, but in general it happens very rarely that they are incompatible. Also u change for 'higher' release to which we most probably will upgrade hence the risk is small.

Reply:
Thanks a lot for the feedback! :) Sounds good to me, so we only need to take care of making customers aware of it.

Reply:
yes, this feature is targeted for 1 customer only for now